python/ray/tests/test_basic.py

```diff
@@ -2432,6 +2432,9 @@ def test_global_state_api(shutdown_only):
     with pytest.raises(Exception):
         ray.nodes()
 
+    with pytest.raises(Exception):
+        ray.jobs()
```

Comment:
This should be changed to make sure it actually raises the error message we expect.

`with pytest.raises(Exception, match="The ray global state API cannot be used before ray.init has been called."):`

Reply:
fixed